### ISSUE TITLE
Add tests for shared memory exports

### ISSUE THREAD

**Issue by ashvardanian** [unum-cloud/UKV#128]
In unit.cpp we must create a child process, read into shared memory and check that the supplied pointed is available from a child process. The normal read must fail in that setting.

**Comment by TonyBhargav** [comment]
hi @ashvardanian  , I would like to work on this issue as a part of course work ,It would be a great opportunity for me to learn about the project architecture and design as well. Could you please assign it to me

**Comment by ashvardanian** [comment]
Yes, I can assign it to you as well. Feel free to propose a solution and join the [discussion group on Discord](https://discord.gg/4mxGrenbNt).

**Comment by TonyBhargav** [comment]
Thank you @ashvardanian , I will look into the documentation
